modules/fedimint-ln-common/src/lib.rs

```diff
@@ -197,13 +200,15 @@ impl LightningGatewayRegistration {
     }
 }
 
-/// Information about a gateway that is shared with other federations and
+/// Information about a gateway that is shared with other federation members and
 /// expires based on a TTL to allow for sharing between nodes with
 /// unsynchronized clocks which can each anchor the announcement to their local
 /// system time.
 #[derive(Debug, Clone, Serialize, Deserialize, Encodable, Decodable, PartialEq, Eq, Hash)]
 pub struct LightningGatewayAnnouncement {
     pub info: LightningGateway,
+    /// Indicates if this announcement has been vetted by the federation
+    pub vetted: bool,
```

Comment:
Not sure if this belongs into the `LightningGatewayAnnouncement` and `LightningGatewayRegistration`, feels like this could easily lead to bugs where gateways can announce themselves as "vetted". As far as I can tell the only thing preventing this right now is that `filter_duplicate_gateways` always sets the field to `false`.

Reply:
Makes sense. Is there a better fix than replacing `LightningGatewayAnnouncement` in `register_gateway` with a different struct? didn't want to add yet another struct for the gateway registration circuit

Reply:
It's ok for now, I don't have a much better idea. Is it really needed for both of the structs? Feels like only the one saved to/loaded from the DB needs it but not the one transmitted over the network. Then the field could default to `false` whenever one is converted to the other,

Reply:
could remove it for `LightningGatewayRegistration` transmitted over the network

Reply:
though when we have dynamic meta, then we'd apply vetted field to the registration struct, and not announcement